src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +53,54 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/**
+ * Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = (element: Element | Document) => {
+  // Find the time elements to act on.
+  const timeElements = [].concat(
+    element instanceof HTMLTimeElement ? [element] : [],
+    Array.from(element.getElementsByTagName('time')),
+  );
+
+  timeElements.forEach(elem => {
+    // Present only because the Flow DOM libdef does not have a specific definition
+    // for `getElementsByTagName('time')`, so the typedef of `timeElements` is wider
+    // than it actually is.
+    if (!(elem instanceof HTMLTimeElement)) {
+      return;
+    }
```

Comment:
Hmm, really? I see this in `dom.js` in my latest /tmp/flow/flowlib_* directory:
```js
  getElementsByTagName(name: 'time'): HTMLCollection<HTMLTimeElement>;
```

Reply:
... Oho. It has that on `Document`... but not on `Element`.

This would be great to report on the Flow issue tracker :slightly_smiling_face: : https://github.com/facebook/flow/issues
Then the comment can just link to the issue.

Reply:
Done. https://github.com/facebook/flow/issues/8450